code/modules/reagents/Chemistry-Reagents/Chemistry-Reagents-Ethanol.dm

```diff
@@ -1559,3 +1559,12 @@
 
 	glass_name = "horchata"
 	glass_desc = "A lovely looking horchata del chufa."
+
+/datum/reagent/ethanol/uristhomebrew
+	name = "Urists Moonshine"
+	description = "Pure 100% distilled plump helmet ethanol"
+	taste_description = "liquid fire, plump helmets"
+	color = "#b31978"
+	strength = 200
```

Comment:
You have successfully made the weakest alcohol imaginable.
For every alcohol strength is how many units of said alcohol it takes to go from one stage of drunkenness to the next.